core/tests/protractor_utils/users.js

```diff
@@ -19,15 +19,17 @@
 
 var general = require('./general.js');
 var waitFor = require('./waitFor.js');
-
+var action = require('./action.js');
 var AdminPage = require('./AdminPage.js');
+const { browser } = require('protractor');
```

Comment:
This is not required, I think.

Reply:
I tested without the import, it is working. I will remove the import.